_drafts/SC1-3-5-autocomplete-valid.md

```diff
@@ -0,0 +1,177 @@
+---
+name: Autocomplete valid
+description: |
+  This rule checks that the HTML autocomplete attribute has a correct value
+
+success_criterion:
+- 1.3.5 # Identify Input Purpose
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Wilco Fiers
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any HTML `input`, `select` and `textarea` element with a [non-empty]() `autocomplete` attribute, except if one of the following is true:
+
+- The element is **not** [[visible on the page]] or [[exposed to assistive technologies]]
+- The element is an `input` element a `type` of `hidden`, `button`, `submit` or `reset`
+- The element has a `disabled` or `aria-disabled="true"` attribute
+- The element has `tabindex="-1"` and has a [[semantic role]] that is not a [widget](https://www.w3.org/TR/wai-aria-1.1/#widget_roles)
+
+### Expectation 1
+
+The `autocomplete` attribute is a single term, or a space seperated list of terms.
+
+### Expectation 2
+
+The autocomplete term(s) follow the [HTML 5.2 specification](https://www.w3.org/TR/html52/sec-forms.html#autofill-detail-tokens), which requires that it/they match the following:
```

Comment:
One thing to consider about this is if we should only care about the last term in the list. I tested in Chrome and it certainly doesn't let you autocomplete if you got any of these things wrong, but our non-existing future AT might be less picky about it. I slightly prefer having it follow the spec. It's easier to lessen the restrictions on this later than it would be to make it loose now and restrict it in the future.